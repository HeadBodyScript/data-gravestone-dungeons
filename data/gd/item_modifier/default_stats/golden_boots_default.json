{
  "function": "minecraft:set_attributes",
  "modifiers": [
    {
      "attribute": "minecraft:generic.armor",
      "name": "",
      "amount": 1,
      "operation": "addition",
      "id": "3162d866-6752-4c8b-a128-e0829ac7695d",
      "slot": "feet"
    }
  ]
}