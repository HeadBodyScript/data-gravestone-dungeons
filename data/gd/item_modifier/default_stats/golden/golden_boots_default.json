[
  {
    "function": "minecraft:set_lore",
    "lore": [
      {
        "text": ""
      },
      {
        "text": "Base stats:",
        "color": "gray",
        "italic": false
      },
      {
        "text": " +1 Armor",
        "color": "blue",
        "italic": false
      },

      {
        "text": ""
      }
    ],
    "mode": "insert"
  },
  {
    "function": "minecraft:set_attributes",
    "modifiers": [
      {
        "attribute": "minecraft:generic.armor",
        "amount": 1,
        "operation": "add_value",
        "id": "6f9de10d-b88d-4948-8471-0dec12250c40",
        "slot": "feet"
      }
    ],
    "replace": false
  }
]